cloudapi/config_test.go

```diff
@@ -0,0 +1,51 @@
+package cloudapi
```

Comment:
Add the copyright header here.

Reply:
Resolved by https://github.com/loadimpact/k6/pull/1881/commits/b89c3441837fd0dcc4cbf1fd48ed30d08532bf2b